packages/back-end/src/services/features.ts

```diff
@@ -44,6 +47,55 @@ function generatePayload(
   return defs;
 }
 
+// TODO Put this in a better place?
+function generateVisualExperimentsPayload(
+  visualExperiments: Array<{
+    experiment: ExperimentInterface;
+    visualChangeset: VisualChangesetInterface;
+  }>,
+  _environment: string,
+  _groupMap: GroupMap
+): SDKExperiment[] {
+  return visualExperiments.map(({ experiment: e, visualChangeset: v }) => {
+    const phase = e.phases.slice(-1)[0];
```

Comment:
Should we have a sanity check to skip experiments if the phases array is empty?

Reply:
Yes - added